src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+const largeImg = document.getElementById('largeImg');
+
+gallery.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const imgLink = e.target.closest('.list-item__link');
+
+  if (!imgLink) {
+    return;
+  }
+
+  largeImg.src = imgLink.href;
```

Comment:
in ur case will be the same, but the shortest
```suggestion
  if (imgLink) {
    largeImg.src = imgLink.href;
  }
```